tests/unit/RestorationApplicantInformation.spec.ts

```diff
@@ -3,7 +3,7 @@ import { RestorationApplicantInformation } from '@/views'
 import { RestorationResources } from '@/resources/'
 
 describe('Restoration Review Confirm view', () => {
-  it('renders the component properly', () => {
+  xit('renders the component properly', () => {
```

Comment:
Does this test not work?

Reply:
It was failing. So uncommented it but I can fix it and commit. Just didn't want to make changes if someone else was parallelly working on it

Reply:
Yes, please put it back. Any idea why it's failing? LMK if I can help.